dashboard/app/controllers/teacher_dashboard_controller.rb

```diff
@@ -1,6 +1,14 @@
 class TeacherDashboardController < ApplicationController
   load_and_authorize_resource :section
 
+  rescue_from CanCan::AccessDenied do
+    if params[:course_name]
```

Comment:
While we are here, do you want to also add the unit page? Totally fine if you want to wait until later

Reply:
Do we need to do anything in the routes file to add the :course_name param? I can't quite remember how we did it